server/controllers/wazuh-api.ts

```diff
@@ -967,14 +964,13 @@ export class WazuhApiCtrl {
       }
       const delay = (data || {}).delay || 0;
       if (delay) {
-        const current = new Date();
-        this.queue.addJob({
-          startAt: new Date(current.getTime() + delay),
-          type: 'request',
-          method,
-          path,
-          data,
-          options,
+        addJobToQueue({
+          startAt: new Date(Date.now() + delay),
+          run: async () => {
+            try{
+              await context.wazuh.api.client.asCurrentUser.request(method, path, data, options);
+            }catch(error){};
```

Comment:
Here we should log a possible error or propagate ?

Reply:
Good idea! 👍